src/dune_rules/jsoo/js_of_ocaml.ml

```diff
@@ -242,6 +358,7 @@ module Env = struct
     ; sourcemap = None
     ; runtest_alias = None
     ; flags = Flags.default ~profile
+    ; enabled_if = Some Blang.true_
```

Comment:
This is so that we don't need a case for `None` there:
https://github.com/ocaml/dune/blob/ec321a5aff5093385a428cbce7b7963fe642acd3/src/dune_rules/jsoo/jsoo_rules.ml#L18-L20
https://github.com/ocaml/dune/blob/ec321a5aff5093385a428cbce7b7963fe642acd3/src/dune_rules/jsoo/jsoo_rules.ml#L603-L605